cmd/root.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2022-2023 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"github.com/veraison/apiclient/management"
+)
+
+var (
+	version = "0.0.1"
+
+	configFile string
+	config     = &Config{}
+
+	rootCmd = &cobra.Command{
+		Short:   "policy management client",
+		Version: version,
+	}
+
+	service *management.Service
+)
+
+type Config struct {
+	Host string
+	Port int
+}
+
+func Execute() {
+	cobra.CheckErr(rootCmd.Execute())
+}
+
+func init() {
+	cobra.OnInitialize(initConfig)
+
+	rootCmd.PersistentFlags().StringVarP(&configFile, "config", "c", "", "config file")
```

Comment:
```suggestion
	rootCmd.PersistentFlags().StringVarP(&configFile, "config", "c", "", "configuration file for pocli tool")
```

Reply:
I think the "for pocli tool" is entirely redundant, given that this is the help message for the too.